src/DetailTile.js

```diff
@@ -1,14 +1,20 @@
 import React from 'react';
 import './DetailTile.css';
 
-const DetailTile = ({ title, posterPath, averageRating, releaseDate, id, seeAllMovies }) => {
+const DetailTile = (props) => {
```

Comment:
Was there a reason you went from destructuring props to not destrcturing here?

Reply:
We went from having another container between App and DetailTile, to getting rid of it. I think that just changed how we passed things down?